doc_build/BUILD

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
```

Comment:
Could all this stuff go into the `docs` directory without getting in the way of what's in github pages?  Maybe also call it `docs_build` to get things to line up more nicely?

Reply:
Alas, no. github tries to publish the source files.